torch/_torch_docs.py

```diff
@@ -2491,20 +2486,15 @@ def merge_dicts(*dicts):
 imag(input, out=None) -> Tensor
 
 Returns the imaginary values of the given :attr:`input` tensor. If
-:attr:`input` is a real (non-complex) tensor this is equivalent to
-``torch.zeros_like(input)``.
+:attr:`input` is a real (non-complex) tensor it has no imaginary part
```

Comment:
the line above also seems out of date.

Reply:
Take a look. I added a warning to try and clarify this while describing the intent of the function generically.